.github/workflows/changelog.yml

```
@@ -0,0 +1,75 @@
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add CHANGELOG.md
+          git commit -m "chore: update changelog with git-cliff
+          
+          Auto-generated changelog using git-cliff based on conventional commits."
+      
+      - name: Push changes
+        if: steps.verify-changed-files.outputs.changed == 'true'
+        run: |
+          git push origin HEAD:${{ github.ref_name }}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)